annotations/views/user_views.py

```diff
@@ -394,3 +402,40 @@ def list_user(request):
         'title': 'Contributors'
     }
     return render(request, template, context)
+
+@staff_member_required
```

Comment:
I think this should be admin. I don't think we really use the staff role at this point, do we?